network.go

```diff
@@ -195,6 +195,12 @@ func (n *network) CreateEndpoint(name string, options ...EndpointOption) (Endpoi
 	if name == "" {
 		return nil, ErrInvalidName(name)
 	}
+
+	e, _ := n.EndpointByName(name)
```

Comment:
I agree we need to check here if the ep already exist. 
But just curious how you hit the problem as bridge should already run this check and return the error.
Was it for the special drivers host/none ?